community/github/pull-requests.md

```diff
@@ -0,0 +1,211 @@
+
+## Pull Requests
+
+New pull requests should:
+
+* be created from a fork rather than the pyOpenSci owned parent repository.
+* never be submitted from the `main` branch of your fork.
+
+Authors of new pull requests should whenever possible, do their best to create clean pull requests. 
+
+A clean pull request:
+
+* is focused, and easy to review,
+* can be more quickly reviewed (and in turn merged) so it saves everyone time.
+* closes an existing issue (with the exception of new content such as blog posts that we manage internally using Asana)
+
+:::{tip}
+Review your own pull request before asking someone else to review it for you. You might be surprised that you notice things in the pull request that you didn't notice when working on the content locally.
+:::
+
+### Pull request content 
+
+Below are some guidelines for pull request content. Clean pull requests lead to simpler reviews & faster merging! 
+
+1. **Keep It Small**: Aim for one logical change per pull request to simplify review.
+1. **Create descriptive PR Title and Summary**: Clearly state what the PR achieves and why, including any related issue numbers or discussions.
+1. **Double check that the files committed to the pull request clearly align with the suggested changes being made in the pull request**. For example if a new guidebook page contains images, all images that are new to the guidebook should be included in the list of files in the pr. If the new page reuses images, then link to the existing images in the repository rather than re-adding them.
+1. **Follow Coding / style guide and other organization Standards (if applicable)**: Stick to the project's coding guidelines for style and organization if they exist. This is more often relevant to code pull requests than text.
+1. **Review Your Own Pull request first**: Look over your PR critically before requesting reviews to catch any mistakes early.
+1. **Respond Respectfully**: Be open to feedback and discuss suggestions to improve  the project.
+1. **Check CI for any red x's (more on CI below)**: If your pull request returns a red X in the "checks/CI" section, then it's worth seeing if you can figure out what is broken in the build. If you aren't sure - no problem. Leave a note for the reviewer to allow them to help you understand what needs to be fixed (if anything).
+
+### Highly Recommended
+
+1. **Create clear Commit Messages**: Explain why changes are made, not just what was changed.
+1. **Keep Your Branch Updated**: Regularly rebase your fork from the main branch (is possible) to avoid / clean up any merge conflicts and to keep your PR up to date.
+
+:::{info}
+* [tips for better pull requests](https://opensource.com/article/18/6/anatomy-perfect-pull-request)
+* pyOpenSci has also enabled a "update branch" feature in the pull request which will update your branch to be in sync with main. It creates a merge commit rather than a rebase in most cases. 
```

Comment:
```suggestion
* pyOpenSci has also enabled an "update branch" feature in the pull request, which will update your branch to be in sync with main. In most cases, this creates a merge commit rather than a rebase. 
```